### ISSUE TITLE
Getting 400 invalid request when triggering gradle sync

### ISSUE THREAD

**Issue by waterone** [facebook/fresco#690]
I am getting this error while building project

Error:Could not GET 'https://raw.githubusercontent.com/layerhq/releases-android/master/releases/com/facebook/fresco/fresco/'. Received status code 400 from server: Bad Request
<a href="toggle.offline.mode">Enable Gradle 'offline mode' and sync project</a>

**Comment by massimocarli** [comment]
The problem is on the server. I got the same accessing from the web.

**Comment by massimocarli** [comment]
How are you importing fresco library? Could you provide more information please?
Thanks

**Comment by waterone** [comment]
@massimocarli
Thanks for replying 

I am  using this in build.gradle
compile 'com.facebook.fresco:fresco:0.8.0+'

Actually the problem occur when I tried to import a swipe to dismiss library which has a version mismatch of build tool . When I removed that library and compiled the project with old build.gradle file fresco works fine.

Hope this information is helpful.
Currently I am not facing the same issue as I am not using the other library.

Thanks for the help.
